src/js/control/textarea.tinymce.js

```diff
@@ -22,7 +22,7 @@ export default class controlTinymce extends controlTextarea {
    * configure the tinymce editor requirements
    */
   configure() {
-    this.js = ['//cdn.tinymce.com/4/tinymce.min.js'];
+    // this.js = ['//cdn.tinymce.com/4/tinymce.min.js'];
```

Comment:
?

Reply:
This took a long time to load so I pulled it out because I already was referencing it in my other project. How can I tell FB to not attempt to load this?